impactcommon/exogenous_economy/gdppc.py

```diff
@@ -62,50 +220,39 @@ def get_timeseries(self, hierid):
         
         return iso_gdppcs * ratio.values[0]
 
+    @lru_cache(maxsize=None)
```

Comment:
LRU cache has to do some sorting on every call, which is wasted time with no maxsize. (And we definitely don't want a max size, since we know there will only ever be ~200 entries). All we really want is a one-step lookup memoization. There must be something simpler for that (or we could make that decorator).

Reply:
@jrising, I don't think that's how this is implemented. When maxsize is None, this is simply unbounded cache: https://github.com/python/cpython/blob/4b47a5b6ba66b02df9392feb97b8ead916f8c1fa/Modules/_functoolsmodule.c#L828 

It's a C-wrapper. It adjusts references (this is how python does garbage collection) and counts hits and misses and that's about it.

`@lru_cache(maxsize=None)` becomes `@cache`, in later releases.

Reply:
Okay, that's fine then.